### ISSUE TITLE
代码问题

### ISSUE THREAD

**Issue by wzr0108** [Plachtaa/seed-vc#30]
inference.py
```
source_audio = source_audio[:sr * 30]
source_audio = torch.tensor(source_audio).unsqueeze(0).float().to(device)

ref_audio = ref_audio[:(sr * 30 - source_audio.size(-1))]
```
sr * 30 - source_audio.size(-1)不是0吗，这里是否有问题

**Comment by Plachtaa** [comment]
ndim=1时0和-1是一样的

**Comment by wzr0108** [comment]
torch==2.2.0+cu118，ref_audio会变成空

**Comment by wzr0108** [comment]
注释ref_audio = ref_audio[:(sr * 30 - source_audio.size(-1))]会对结果有影响吗
